### DISCUSSION TITLE
Unable to cancel individual copilot subscription

### DISCUSSION THREAD

**Discussion by anand-practice** [community/community#74700] [New to GitHub] [answered]
Hey guys, I'm unable to cancel github copilot subscription, in "Billing & Plans -> Plans & usage" section I cannot see the "Manage Subscription" or "Subscription" to cancel it. I had an active subscription for last 6 months but payment failed this month, but still I want to cancel subscription but unable to.

**Comment by marielenna** [comment, marked as answer]
Hi there! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Comment by fellan1996** [comment]
That is probably because Microsoft is a evil company

**Comment by kronkinatorix** [comment]
If copilot was a person I'd force it to eat glass and enjoy every moment of it.

**Reply by cognivore** [reply]
You are a very disturbing person, your message made me deeply sad.

**Comment by MichaelKMorrison** [comment]
From https://github.com/settings/billing/summary scroll to Add-ons then in the GitHub Copilot section there is a mana subscriptions option.
![image](https://github.com/community/community/assets/15235626/16aa5707-802c-4980-80c7-6268e0150cf0)

**Reply by ydhnwb** [reply]
Doesnt work on my case, I stopped the subscription on april 7th, and they still try to charges me until today.

![image](https://github.com/community/community/assets/26734262/48d5365a-e357-4cf4-9b34-de70f1ccf624)

I created a support ticket for this and see if it helps

**Reply by oapapas** [reply]
Can you provide us an update on what happened?

**Reply by oapapas** [reply]
I had a similar issue. I opened a support ticket, and I got this response to my ticket:

It looks like your Copilot refund request may be eligible to be handled by our virtual assistant which would get this processed for you immediately.
Can you please try following these steps?:
Follow this specific link to our contact form: [https://support.github.com/contact/account?flow=refund_copilot_individual]
Scroll down a bit to see a blue button activated to "Try our virtual agent".
Follow the prompts to request the cancellation and prorated _refund.

I did this and worked fine.

**Reply by zhou96-gh** [reply]
2024年5月24日取消订阅，当天依旧进行了退款并且未提供相应的服务

并且支付单显示
日期:2024年5月24日太平洋时间晚上08:53
服务截止日期:2024-06-01

**Reply by zhou96-gh** [reply]
提交工单2天并未得到任何回复

**Reply by zhou96-gh** [reply]
> 我遇到了类似的问题。我打开了支持单，并收到了以下回复：
> 
> 您的 Copilot 退款请求似乎可以由我们的虚拟助理处理，虚拟助理会立即为您处理。 您可以尝试按照以下步骤操作吗？： 按照此特定链接访问我们的联系表：[https://support.github.com/contact/account?flow=refund_copilot_individual] 向下滚动一点，即可看到激活的蓝色按钮“尝试我们的虚拟代理”。 按照提示请求取消和按比例退款。
> 
> 我这样做了并且效果很好。

我操作提示没有任何订阅服务，但实际已出现扣款
